sdk/keyvault/azkeys/example_test.go

```diff
@@ -21,7 +21,7 @@ var client *azkeys.Client
 
 func ExampleNewClient() {
 	vaultUrl := os.Getenv("AZURE_KEYVAULT_URL")
-	cred, err := azidentity.NewDefaultAzureCredential(nil)
+	cred, err := azidentity.NewClientSecretCredential("myTenantID", "myClientID", "myClientSecret", nil)
```

Comment:
Why was this changed?  I'm not pushing back, but I thought that we wanted to promote `DefaultAzureCredential`.